pymc3/distributions/logprob.py

```diff
@@ -141,8 +130,88 @@ def logpt(
         A scaling term to apply to the generated log-likelihood graph.
     transformed
         Apply transforms.
-    cdf
-        Return the log cumulative distribution.
+
+    """
+    if not isinstance(rv_values, Mapping):
+        rv_values = (
+            {var: at.as_tensor_variable(rv_values).astype(var.type)}
+            if rv_values is not None
+            else {}
+        )
+
+    tmp_rv_values = {}
+    transform_map = {}
+    for node in io_toposort(graph_inputs((var,)), (var,)):
+        if isinstance(node.op, RandomVariable):
+            curr_var = node.out
+            rv_value_var = getattr(
+                curr_var.tag, "observations", getattr(curr_var.tag, "value_var", curr_var)
+            )
+            rv_value = rv_values.get(curr_var, rv_value_var)
+            tmp_rv_values[curr_var] = rv_value
+
+            if hasattr(rv_value_var.tag, "transform") and transformed:
+                transform_map[rv_value] = rv_value_var.tag.transform
+
+    transform_opt = TransformValuesOpt(transform_map)
+    logp_var = joint_logprob(tmp_rv_values, sum=sum, extra_rewrites=transform_opt, **kwargs)
+
+    if transform_map and jacobian:
+        for transform_rv_val, transform in transform_map.items():
+            # Following line gets keys corresponding to given value in a dictionary
+            transform_rv = list(tmp_rv_values.keys())[
+                list(tmp_rv_values.values()).index(transform_rv_val)
+            ]
+            tr_inputs = transform_rv.owner.inputs if transform_rv.owner else []
+            transformed_jacobian = transform.log_jac_det(transform_rv_val, *tr_inputs)
+            if transformed_jacobian:
+                if logp_var.ndim > transformed_jacobian.ndim:
+                    logp_var = logp_var.sum(axis=-1)
+                logp_var += transformed_jacobian
+
+    if scaling:
+        logp_var *= _get_scaling(
+            getattr(var.tag, "total_size", None), rv_value.shape, rv_value.ndim
+        )
+
+    # Recompute test values for the changes introduced by the replacements
+    # above.
+    if config.compute_test_value != "off":
+        for node in io_toposort(graph_inputs((logp_var,)), (logp_var,)):
+            compute_test_value(node)
+
+    if var.name is not None:
+        logp_var.name = "__logp_%s" % var.name
+
+    return logp_var
+
+
+def logcdfpt(
+    var: TensorVariable,
+    rv_values: Optional[Union[TensorVariable, Dict[TensorVariable, TensorVariable]]] = None,
+    *,
+    jacobian: bool = True,
+    scaling: bool = True,
+    transformed: bool = True,
+    sum: bool = False,
```

Comment:
```suggestion
    scaling: bool = True,
    sum: bool = False,
```

I don't think those 2 ever make sense for cdfs